.github/workflows/fuzzers.yml

```diff
@@ -21,17 +21,17 @@ jobs:
           - aflcc
           - aflfast
           - aflplusplus
-          - aflplusplus_optimal
-          - aflplusplus_eclipser
+          # Disabled due to segfaults and timeouts.
+          # See https://github.com/google/fuzzbench/issues/1073.
+          # - aflplusplus_optimal
```

Comment:
needs also removal in core fuzzers perhaps?

Reply:
Yes done.